torch/_inductor/codecache.py

```diff
@@ -485,11 +485,7 @@ def _load_library(path):
     @classmethod
     def load(cls, source_code):
         picked_vec_isa = pick_vec_isa()
-        key, input_path = write(
-            source_code,
-            "cpp",
-            extra=cpp_compile_command("i", "o", vec_isa=picked_vec_isa),
-        )
+        key, input_path = write(source_code, "cpp")
```

Comment:
We need to include the `cpp_compile_command` in the hash so we recompile if it changes.

Reply:
included `code_hash(repr(cpp_compile_command(...)))` as part of the hash